### PR TITLE
Remove /v1/unit_data from SDS_API_BASE_URL

### PR DESCRIPTION
**Pull request by kylelawsonAND**
### What is the context of this PR?
The base url used for making requests to SDS was previously configured to not be the base, but the end point `v1/unit_data`, preventing any future requests to other endpoints from using the env var which details this url. 
This PR fixes this by ensuring the env var is actually the base url of SDS and that endpoints are appended to. the base when needed by runner.

### Changes
**.development.env & .functional-tests.env**: `SDS_API_BASE_URL` now set to `http://localhost:5003`, having had` /v1/unit_data` removed from the end of it.

**session.py**: Imports and uses the renamed method `get_supplementary_data_v1`.

**supplementary_data.py**: Renamed method `get_supplementary_data` to `get_supplementary_data_v1`, as it now appends `/v1/unit_data` to the SDS API Base URL. 

**doc/run-mock-sds-endpoint.md**: Fixed reference to the env var `SDS_API_BASE_URL` to ensure it instructs the reader to correctly set it to the base url. 

**test_request_supplementary_data.py**: Renamed usages of `get_supplementary_data` to `get_supplementary_data_v1`

### How to review
Ensure all tests are still passing.
Run Runner, Launcher and the Mock SDS and ensure mock data can still be loaded.

### CODE REVIEW

**Comment by kylelawsonAND** [comment]
[Runner SDS base URL shouldn't include endpoint names (1 SP)](https://trello.com/c/lfrcFth8/5777-runner-sds-base-url-shouldnt-include-endpoint-names-1-sp)

File: doc/run-mock-sds-endpoint.md [hunk 1]
```diff
@@ -4,20 +4,25 @@ In order to test loading supplementary data, we have a development script that c
...
-SDS_API_BASE_URL=http://localhost:5003/v1/unit_data
+SDS_API_BASE_URL=http://localhost:5003
 ```
 
 From the home directory, run using
+
 ```python
-python -m scripts.mock_sds_endpoint
+python - m
+scripts.mock_sds_endpoint
```
Comment (katie-gardner) [inline doc/run-mock-sds-endpoint.md:16]: this change looks accidental

**Review by katie-gardner — CHANGES_REQUESTED** [review]
All working nicely, just one comment

File: doc/run-mock-sds-endpoint.md (hunk 1 above)
Comment (MebinAbraham) [reply doc/run-mock-sds-endpoint.md:16]: Will be good to get this in, so happy for you to make this change. We'll get a 2nd ✅ from Yuyu.

Comment (katie-gardner) [reply doc/run-mock-sds-endpoint.md]: done - and fixed language for the snippet

**Review by Yuyuutsu — APPROVED** [review]
Comment resolved and everything else lgtm !

**Review by katie-gardner — APPROVED** [review]

File: doc/run-mock-sds-endpoint.md (hunk 1 above)
Comment (MebinAbraham) [reply doc/run-mock-sds-endpoint.md]: Looks good, happy to resolve.

**Pull request merged**